src/apis.ts

```diff
@@ -26,6 +78,76 @@ export async function fetchPlayerData(
 
   return resp.data.player as HypixelPlayerResponse;
 }
+export async function fetchBedwarsData(
+  uuid: string
+): Promise<PlayerStatsResponse | null> {
+  let resp: AxiosResponse;
+  try {
+    resp = await axios({
+      url: "player",
```

Comment:
```suggestion
      url: "https://api.hypixel.net/player",
```
baseUrl is only used in configs

Reply:
Done (changed the other one too)